packages/arb-token-bridge-ui/src/token-bridge-sdk/teleport.ts

```diff
@@ -0,0 +1,64 @@
+// utils for teleport type transactions
+
+import { Erc20L1L3Bridger, EthL1L3Bridger, getL2Network } from '@arbitrum/sdk'
+import { getProviderForChainId } from './utils'
+import { TELEPORT_ALLOWLIST, isNetwork } from '../util/networks'
+import { Provider } from '@ethersproject/providers'
+
+export const isTeleport = ({
+  sourceChainId,
+  destinationChainId
+}: {
+  sourceChainId: number
+  destinationChainId: number
+}) => {
+  const isSourceChainEthereum =
```

Comment:
tbh we don't need these checks because they are inferred when accessing `TELEPORT_ALLOWLIST`. We define L1 keys and L3 values, so it's always L1 source and L3 destination.

The entire method could just be:
```
return TELEPORT_ALLOWLIST[sourceChainId]?.includes(destinationChainId)
```

Reply:
Good point, now that we are maintaining a hardcoded list we don't need other checks. Updated.